.changeset/itchy-boxes-try.md

```diff
@@ -0,0 +1,6 @@
+---
+'firebase': minor
```

Comment:
@hsubox76 Do you think this should be a minor or patch bump?

Reply:
Let's make it a minor, technically correct and gives people a heads up about a type change.